app/views/users/_form.html.haml

```diff
@@ -26,10 +26,10 @@
     - @partner_organisations.each do |dp|
       = f.govuk_check_box :additional_organisations, dp.id, label: { text: dp.name }, checked: @user.additional_organisations.include?(dp)
 
-  = f.govuk_collection_radio_buttons :active,
-      user_active_options,
-      :id,
-      :name,
-      legend: { tag: :h2 }
-
-  = f.govuk_submit t("form.button.user.submit")
+  .govuk-button-group
+    = f.govuk_submit t("form.button.user.submit")
+    - if !@user.new_record?
```

Comment:
Oh, darn, forgot to say you could have used the antonym `if @user.persisted?` here

Reply:
L'esprit d'escalier